### ISSUE TITLE
t.toUpperCase is not a function - prisma 2.8.0

### ISSUE THREAD

**Issue by Sly777** [prisma/studio#556]
I'm getting an error when I click any ENUM row in the studio. And also it shows red somehow. It started to happen after getting the 2.8.0 update.

![image](https://user-images.githubusercontent.com/694940/94913374-71ca4e80-04a9-11eb-9009-14db72411a2e.png)

1. Prisma version (`prisma -v` or `npx prisma -v`):
```
@prisma/cli          : 2.8.0
Current platform     : darwin
Query Engine         : query-engine e6c9b4b2b7fa162d0d459d1863321f547498fcfe (at node_modules/@prisma/cli/query-engine-darwin)
Migration Engine     : migration-engine-cli e6c9b4b2b7fa162d0d459d1863321f547498fcfe (at node_modules/@prisma/cli/migration-engine-darwin)
Introspection Engine : introspection-core e6c9b4b2b7fa162d0d459d1863321f547498fcfe (at node_modules/@prisma/cli/introspection-engine-darwin)
Format Binary        : prisma-fmt e6c9b4b2b7fa162d0d459d1863321f547498fcfe (at node_modules/@prisma/cli/prisma-fmt-darwin)
Studio               : 0.294.0
```

2. Logs from Developer Tools Console or Command line, if any:
```
TypeError: t.toUpperCase is not a function
    at http://localhost:5555/studioBundle.js:395:233946
    at Array.filter (<anonymous>)
    at Proxy.forEach.fn.<computed> (http://localhost:5555/studioBundle.js:16:38493)
    at vu.getMatches (http://localhost:5555/studioBundle.js:395:233934)
    at vu.render (http://localhost:5555/studioBundle.js:395:234621)
    at _e (http://localhost:5555/studioBundle.js:16:9807)
    at http://localhost:5555/studioBundle.js:395:74880
    at le (http://localhost:5555/studioBundle.js:16:7913)
    at e.track (http://localhost:5555/studioBundle.js:16:19256)
    at vu.l [as render] (http://localhost:5555/studioBundle.js:395:74828)
    at vu.pt (http://localhost:5555/studioBundle.js:395:74976)
    at vu.t.render (http://localhost:5555/studioBundle.js:395:74214)
    at Gs (http://localhost:5555/studioBundle.js:379:69874)
    at ks (http://localhost:5555/studioBundle.js:379:69669)
    at va (http://localhost:5555/studioBundle.js:379:105468)
    at ul (http://localhost:5555/studioBundle.js:379:96715)
    at al (http://localhost:5555/studioBundle.js:379:96640)
    at Za (http://localhost:5555/studioBundle.js:379:93670)
    at http://localhost:5555/studioBundle.js:379:45312
    at t.unstable_runWithPriority (http://localhost:5555/studioBundle.js:387:3844)
    at Br (http://localhost:5555/studioBundle.js:379:45021)
    at Yr (http://localhost:5555/studioBundle.js:379:45257)
    at qr (http://localhost:5555/studioBundle.js:379:45192)
    at Ka (http://localhost:5555/studioBundle.js:379:90455)
    at Object.enqueueForceUpdate (http://localhost:5555/studioBundle.js:379:49294)
    at t.w.forceUpdate (http://localhost:5555/studioBundle.js:371:1383)
    at http://localhost:5555/studioBundle.js:387:11590
    at o (http://localhost:5555/studioBundle.js:395:56313)
```

3. Does the issue persist even after updating to the latest `prisma` alpha? (`npm i -D @prisma/cli@dev`)

`I'm using latest prisma but not alpha or dev version`

4. Prisma schema (if relevant):
```

```

**Comment by ahrbil** [comment]
this is the same issue #555 I'm getting.
did you get around this?

**Comment by Sly777** [comment]
Nope unfortunately @ahrbil

**Comment by madebysid** [comment]
Hey everyone, sorry about the delay, I was on vacation.

It seems as though Studio doesn't recognize the value in this cell as an enum variant, so it treats it as invalid (hence the red background). The crash is also somewhat related to it. I've managed to figure out what the issue is and it is fixed in Prisma CLI `v2.9.0-dev.43`. If you update to this version, you should be able to see and edit enums as before. I'd appreciate it if one of you could try this out and let me know if this is still an issue!

This will also go out with the 2.9.0 release next Tuesday in case you're hesitant to use a `dev` version.

**Comment by nickreynke** [comment]
@madebysid Works like a charm! :) Thank you!

**Comment by ahrbil** [comment]
@madebysid also works for me.
thank you all 🙏

**Comment by madebysid** [comment]
Awesome! We also just released 2.8.1 today, so you should be able to update to a stable version rather than use a 'dev' version!
